scenario/context/CometContext.ts

```diff
@@ -272,9 +259,9 @@ export class CometContext {
     const queueEvent = queueTxn.events?.find(event => event.event === 'ProposalQueued');
     await this.setNextBlockTimestamp(queueEvent?.args?.eta.toNumber() + 1);
 
-    // Execute proposal
+    // Execute proposal (w/ gas limit so we see if exec reverts, not a gas estimation error)
```

Comment:
Oh wow, this is going to greatly improve debugging

Reply:
Yeah now it says "Execution reverted with Timelock" instead of UNPREDICTABLE GAS LIMIT 😸  But overall debugging should be a lot simpler now with some of the latest improvements to tracing